homeassistant/components/modbus/climate.py

```diff
@@ -66,6 +74,7 @@
         vol.Optional(CONF_MAX_TEMP, default=35): cv.positive_int,
         vol.Optional(CONF_MIN_TEMP, default=5): cv.positive_int,
         vol.Optional(CONF_STEP, default=0.5): vol.Coerce(float),
+        vol.Optional(CONF_STRUCTURE, default=">f"): cv.string,
```

Comment:
Please create a constant for this default value e.g. `DEFAULT_STRUCTURE` or `DEFAULT_STRUCTURE_PREFIX`